packages/round-manager/src/features/common/FormWizard.tsx

```diff
@@ -6,9 +6,14 @@ export interface FormWizardProps {
   initialCurrentStep?: number;
   // eslint-disable-next-line @typescript-eslint/ban-types
   initialData?: object;
+  configuration?: object;
```

Comment:
Is the configuration data actually free from or is there an expected shape and content? If so, it would be better to specify it, otherwise we're not allowing TypeScript to help us.